### PR TITLE
FHIR-33343 (TG263), FHIR 32827(Genetic Specimen Type additions), 

### PR DESCRIPTION
**Pull request by saulakravitz**
FHIR-33343:  The Radiotherapy Treatment Location and Qualifier Value sets have been reworked to align with AAPM TG263  https://www.aapm.org/pubs/reports/RPT_263.pdf.     Some of the TG263 concepts lack SNOMED codes/qualifiers.   UMLS codes have been used where available, and local codes defined where not available.

FHIR 32827:  Three values added to Genetic Specimen Type value set

### CODE REVIEW

**Review by markkramerus — CHANGES_REQUESTED** [review]

File: input/fsh/VS_Genomics.fsh [hunk 1]
```diff
@@ -139,3 +139,16 @@ Description:    "The type of specimen analyzed in a genetic test. The values are
 * SPTY#TUMOR     "Tumor"
 * SPTY#VITF      "Vitreous Fluid"
 * SPTY#WND       "Wound"
+* SPTY#STL      "Stool = Fecal"
+* SPTY#UR       "Urine"
+* SPTY#ORH      "Other"
```
[the remark this answers is not in the document]
Comment (saulakravitz) [reply input/fsh/VS_Genomics.fsh:144]: Value set binding is already extensible, so OTHER will be removed.

File: input/fsh/VS_Radiotherapy.fsh [hunk 2]
```diff
@@ -128,12 +129,21 @@ Description: "Various modifiers that can be applied to body locations where radi
...
 * SCT#18769003   "Adjacent" // "Juxta-posed (qualifier value)"
-* UMLS#C0687118  "Detached"  // standing in for non-adjacent  (USCRS-33144)
-* include codes from system SCT where concept is-a #258399006  "Lymph node level value (qualifier value)"    
+* SCT#260235002 "Core (qualifier value)"
+* SCT#410679008 "Surface (qualifier value)"
+* SCT#260521003 "Internal (qualifier value)"
+* SCT#261074009 "External (qualifier value)"
+* SCT#261115001 "Laryngeal (qualifier value)"
+* SCT#260548002 "Oral (qualifier value)"
+* SCT#55561003 "Active (qualifier value)"
```
[the remark this answers is not in the document]
Comment (saulakravitz) [reply input/fsh/VS_Radiotherapy.fsh:144]: Active is used to qualify 'Bone marrow structure (body structure)'.  
I don't know the difference between active and inactive.  I can find the difference between red/yellow bone marrow.
Will ask Su.

[the remark this answers is not in the document]
Comment (saulakravitz) [reply input/fsh/VS_Radiotherapy.fsh]: Active dropped as qualifier.
Active Bone Marrow --> Red Bone Marrow (no qualifier).

File: input/fsh/VS_Radiotherapy.fsh [hunk 3]
```diff
@@ -128,12 +129,21 @@ Description: "Various modifiers that can be applied to body locations where radi
 * SCT#40415009    "Proximal (qualifier value)"
 * SCT#255503000    "Entire (qualifier value)"
 * SCT#255609007    "Partial (qualifier value)"
+* SCT#7771000    "Left (qualifier value)"
```
[the remark this answers is not in the document]
Comment (saulakravitz) [reply input/fsh/VS_Radiotherapy.fsh:132]: This had already been merged into master, but not into this branch.

[the remark this answers is not in the document]
Comment (saulakravitz) [reply input/fsh/VS_Radiotherapy.fsh]: Actually, it was fixed on master, and then messed up on this branch.   
I built the list of qualifiers form the spreadsheet, which mixed both types of qualifier.
Now fixed.

**Review by markkramerus — APPROVED** [review]

**Pull request merged**